themes/bootstrap5/templates/Helpers/pagination.phtml

```diff
@@ -28,28 +50,31 @@
                 'page' => $this->previous,
                 'liClasses' => ['page-prev'],
                 'label' => $this->translate('page_prev'),
-                'contentHtml' => $this->icon('page-prev') . ' ' . $this->transEsc('Prev'),
+                'contentHtml' => $this->icon('page-prev') . ' <span class="page-item-text">' . $this->transEsc('Prev') . '</span>',
             ]
           )
         ?>
       <?php endif; ?>
 
-      <?php foreach ($this->pagesInRange as $page): ?>
-        <?=
-          $this->render(
-              'Helpers/pagination-item.phtml',
-              [
-                'params' => $this->params,
-                'current' => $this->current,
-                'page' => $page,
-                'label' => $this->translate('page_num', ['%%page%%' => $page]),
-                'contentHtml' => $this->escapeHtml($page),
-            ]
-          )
-        ?>
-      <?php endforeach; ?>
+      <?php if (count($this->pagesInRange) > 1): ?>
```

Comment:
Is this extra if statement needed? If count is 0, the loop shouldn't iterate anyway...

Reply:
Note that it is `>1` and not `>=1` or `>0?, so it would iterate over one element. 

I also took that change from here https://github.com/vufind-org/vufind/blob/6495f63ce7f0923c63d0e692b7ba462945ff3782/themes/bootstrap5/templates/search/pagination.phtml#L55

Reply:
You can probably tell that I made this review late in the day and was not at my sharpest. ;-)